frc1388-2024/src/main/java/frc/robot/commands/DeployIntakeCommand.java

```diff
@@ -26,12 +31,26 @@ public DeployIntakeCommand(IntakeSubsystem intakeSubsystem, TransitionSubsystem
 
   // Called when the command is initially scheduled.
   @Override
-  public void initialize() {}
-
+  public void initialize() {
+    m_deployIntakeTimer.restart();
+  }
+  
   // Called every time the scheduler runs while the command is scheduled.
   @Override
-  public void execute() {
-    m_intakeSubsystem.setLifterMotor(IntakeConstants.LIFTER_MOTOR_SPEED_DOWN);
+  public void execute() { 
+    if (m_deployIntakeTimer.get() < IntakeConstants.LIFTER_MOTOR_TIME_DOWN) {
+      m_intakeSubsystem.setLifterMotor(IntakeConstants.LIFTER_MOTOR_SPEED_DOWN);
+    } 
+    else if (m_deployIntakeTimer.get() < IntakeConstants.LIFTER_MOTOR_TIME_DOWN + 0.1) {
+      m_intakeSubsystem.setLifterMotor(0); 
+    }
+    else if (m_deployIntakeTimer.get() < IntakeConstants.LIFTER_MOTOR_TIME_DOWN + 0.8) {
```

Comment:
These should be IntakeConstants instead of `+ 0.1` etc.  Especially since these constants are used in other places.